addons/actions/action.js

```diff
@@ -1,6 +1,6 @@
 import {action as addonAction} from '@storybook/addon-actions';
 
-const action = (eventName, include = ['type']) => {
+function action(eventName, include = ['type']) {
```

Comment:
As I was mentioned, could we just remove prettier from this repo?
I don't think it's a good idea overriding enact lint rules for this repo.